packages/suite/src/components/wallet/TransactionItem/components/TransactionHeading.tsx

```diff
@@ -28,11 +28,11 @@ const HeadingWrapper = styled.div`
 
 const ChevronIconWrapper = styled.div<{ show: boolean; animate: boolean }>`
     display: flex;
-    visibility: ${({ show }) => (show ? 'visible' : 'hidden')};
-    margin-left: ${({ animate }) => (animate ? '5px' : '2px')};
+    /* visibility: ${({ show }) => (show ? 'visible' : 'hidden')}; */
```

Comment:
remove instead of commenting out?

Reply:
22598792797b8c4022b75724baf2ef5f6bef4842